dspace/config/dspace.cfg

```diff
@@ -1542,6 +1542,10 @@ solr-database-resync.time-until-reindex = 600000
 # Keep in mind, changing the schedule requires rebooting your servlet container, e.g. Tomcat.
 solr-database-resync.cron = 0 15 2 * * ?
 
+## PROCESS CLEANER SCRIPT
+# Defines how many days to go back from the current date to find out which old processes to delete (default is 14)
+# process-cleaner.days = 14
```

Comment:
Tiny thing, but I'd prefer if we could make this it's own "section" as it currently appears like it's part of the "solr-database-resync" settings.  Maybe update this as follows (this includes some minor cleanup to the description):
```
#----------------------------------------------------------#
#----------PROCESS CLEANER SCRIPT CONFIGURATION------------#
#----------------------------------------------------------#
# Processes older than this number of days will be deleted when the "process-cleaner" script is next run. Default is 14 (i.e. processes that are two weeks or older will be deleted)
# process-cleaner.days = 14
```

This is very minor obviously and doesn't impact the code of this PR.

Reply:
@LucaGiamminonni : Could I get the above minor update made to the `dspace.cfg` in this PR?  That way I can get this PR merged.  Thanks!

Reply:
Thanks @LucaGiamminonni !  Looks good & I'll get this merged then